lark-env.sh

```
@@ -0,0 +1,260 @@
+  . ./$(script_name) boe --boe-env-name boe_bitable_openapi_v2
+  . ./$(script_name) off
+
+Notes:
+  - Use '.' (or 'source' in bash/zsh) to apply proxy env vars in current shell.
+  - The script always updates Whistle rule '${RULE_NAME}'.
+USAGE
+}
+
+fail() {
+  printf '%s\n' "[lark-cli-env] $*" >&2
+  return 1 2>/dev/null || exit 1
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)